### ISSUE TITLE
System Navigation Manager implemented in UWP, Android, IOS, but NOT in WINUI3

### ISSUE THREAD

**Issue by cesarchefinho** [unoplatform/uno#8517]
### Current behavior

System Navigation Manager is not implemented in Winui3, this prevent migrate my projects frow Uno uwp version to uno winui version and handle hardware back button in winui3

### Works on UWP/WinUI

No

### Affected platforms

Windows

### IDE

Visual Studio 2022

**Comment by jeromelaban** [comment]
Thanks for the report. Can you specify which platform you are talking about for WinUI 3? Is it WinAppSDK on Windows?

**Comment by cesarchefinho** [comment]
Yes. SystemNavigationManager is not implemented in winappsdk (windows) but is implemented in Android and iOS and webasm Winui3 plataforms
